### PR TITLE
refactor(editor): create ndv store

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

File: packages/editor-ui/src/Interface.ts [hunk 1]
```diff
@@ -936,49 +935,51 @@ export interface TargetItem {
 	outputIndex: number;
 }
 
-export interface IUiState {
-	sidebarMenuCollapsed: boolean;
-	modalStack: string[];
-	modals: {
-		[key: string]: IModalState;
-	};
+export interface NDVState {
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/Interface.ts:938]: Nitpick: Should start with an _I_ (`INDVState`)  like other interfaces.

**Review by MiloradFilipovic — APPROVED** [review]
Looks good. Have only one nitpick.

File: packages/editor-ui/src/Interface.ts (hunk 1 above)
Comment (mutdmour) [reply packages/editor-ui/src/Interface.ts:938]: I think this is something we should start to move away from..

Comment (mutdmour) [reply packages/editor-ui/src/Interface.ts:938]: Ivan is big on getting rid of this

**Comment by janober** [comment]
Got released with `n8n@0.200.0`

**Pull request merged**